index.js

```diff
@@ -65,5 +63,5 @@ function versionIncluded(nodeVersion, specifierValue) {
 var data = require('./core.json');
 
 module.exports = function isCore(x, nodeVersion) {
-	return has(data, x) && versionIncluded(nodeVersion, data[x]);
+	return Object.prototype.hasOwnProperty.call(data, x) && versionIncluded(nodeVersion, data[x]);
```

Comment:
this would make the code more brittle, since it would break `delete Object.prototype.hasOwnProperty` were ran, or `delete Function.prototype.call`.